src/wlserver.cpp

```diff
@@ -1111,6 +1113,26 @@ void wlserver_touchmotion( double x, double y, int touch_id, uint32_t time )
 	bump_input_counter();
 }
 
+
+void wlserver_capture_touch_down( int touch_id )
```

Comment:
Do we really need these 3 helper functions? They seem small and simple enough to be inlined.